lib/cluster-providers/ec2-cluster-provider.ts

```diff
@@ -84,8 +85,17 @@ export class EC2ClusterProvider implements ClusterProvider {
     readonly options: EC2ProviderClusterProps;
 
 
-    constructor(options?: EC2ProviderClusterProps) {
-        this.options = options ?? { version: KubernetesVersion.V1_19 };
+    constructor(scope?: Construct, options?: EC2ProviderClusterProps) {
+        if (options != null)
+        {
```

Comment:
`if (options != null) { ` , we don't use C++ style for {}

Reply:
This has been resolved.